### DISCUSSION TITLE
FR: Add a config option to disable writing `.next/export-detail.json` when running `next export`

### DISCUSSION THREAD

**Discussion by gregmagolan** [vercel/next.js#43363] [Ideas]
### Describe the feature you'd like to request

A config option to disable writing `.next/export-detail.json` when running `next export`.

When [building Next.js under Bazel](https://github.com/aspect-build/bazel-examples/tree/main/next.js), the `.next` folder is a write-protected input directory to the `next export` target so Bazel prevents Next.js from writing the `.next/export-detail.json` file there which fails the build.

This is relevant to non-Bazel use cases as well in workflows where the `.next` folder is read-only: https://github.com/vercel/next.js/issues/10137

### Describe the solution you'd like

A config option to disable writing `.next/export-detail.json` when running `next export`.

### Describe alternatives you've considered

No alternatives available.

**Comment by gregmagolan** [comment]
cc @tannerwelsh @alexeagle

**Comment by josephglanville** [comment]
How does this change in Next 14? Now that export is no longer a separate command?

**Comment by sebastianvoss** [comment]
It is still an issue resulting in an `EACCES` error. Are there any workarounds available to prevent this?

```
[Error: EACCES: permission denied, open '[...]/bazel-out/darwin_arm64-fastbuild/bin/sample/.next/trace'] {
  errno: -13,
  code: 'EACCES',
  syscall: 'open',
  path: '[...]/bazel-out/darwin_arm64-fastbuild/bin/sample/.next/trace'
}
```
